src/main/resources/assets/cctweaks/lua/prebios-string.lua

```diff
@@ -71,4 +87,12 @@ if native_getfenv then
 	end
 end
 
-return fun()
+-- Try to run the bios, print possible errors
+local _, ok, err = pcall(fun)
```

Comment:
The file not loading is handled above on line 36. I'm pretty sure this will error if `fun` doesn't return anything (which is probably most of the time).